jwala-webapp/src/main/webapp/resources/js/react/group-operations/WebServerControlPanelWidget.js

```diff
@@ -134,7 +137,23 @@ var WebServerControlPanelWidget = React.createClass({
     statics: {
         getReactId: function(dom) {
             return $(dom).attr("data-reactid");
-        }
+        },
+        parseUrlProtocol: function(url) {
+            try {
+                let protocol = url.split("/")[0].toLowerCase();
+                if (protocol.indexOf(WebServerControlPanelWidget.HTTP_PROTOCOL) > -1) {
```

Comment:
Since this is testing for "http:", isn't it possible that "https" won't be returned either if the location.protocol is "http" too?